integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -924,13 +930,24 @@ private void createNewCluster() {
     Path configScript = Paths.get(RESOURCE_DIR, "python-scripts", "introspect_version_script.py");
     executeWLSTScript(configScript, wlstPropertiesFile.toPath(), domainNamespace3);
 
+    List<String> clusterNames = new java.util.ArrayList<>();
+    clusterNames.add(cluster2Name);
+    ClusterList clusters = listClusterCustomResources(domainNamespace3);
+    for (String clusterName : clusterNames) {
```

Comment:
same comment as in integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java

Reply:
done